File: domain/model/src/main/java/co/com/s/model/unitmeasure/UnitMeasure.java
package co.com.s.model.unitmeasure;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
@NoArgsConstructor
@AllArgsConstructor
@Data
@Builder(toBuilder = true)
public class UnitMeasure {
    private String id;

    private Integer code;

    private String name;

    private Integer codeUnitMeasureType;

    private String acronyms;

    private Boolean active;

    private String companyNit;

    private String createdBy;

    private LocalDateTime createdAt;

    private String modifiedBy;

    private LocalDateTime modifiedAt;

    private String unitMeasureType;
}

File: ejemplo1/src/app/Modelo/UnitMeasure.ts
import { NodeCompatibleEventEmitter } from "rxjs/internal/observable/fromEvent";

export class UnitMeasure {
   id:string;
   code:number;
   name:string;
   codeUnitMeasureType:number;
   acronyms: string;
   active: boolean;
   companyNit:String;
   createdBy: string;
   modifiedBy: string;
   unitMeasureType: string;
   
   constructor(id: string, code:number, name:string, codeUnitMeasureType:number,acronyms:string, active:boolean,companyNit:string, createdBy: string,modifiedBy: string,unitMeasureType: string) {
        this. id=id; 
        this.code=code;
        this.name=name;
        this.codeUnitMeasureType=codeUnitMeasureType;
        this.acronyms=acronyms;
        this.active=active;
        this.companyNit=companyNit;
        this.createdBy=createdBy
        this.modifiedBy=modifiedBy;
        this.unitMeasureType=unitMeasureType;
      }
    }
   